index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
+    <link href="css/style.css" rel="stylesheet">
```

Comment:
You've linked the CSS file to your HTML file, but CSS is still empty. Kindly finish styling before making a PR, it would make it possible to evaluate your code's outcome.